README.md

```diff
@@ -1,10 +1,93 @@
 # Aind.Behavior.Pirouette
 
-A repository with source code for long-term, freely moving, electrophisiology recordings in mice
```

Comment:
I would keep a high-level description of the repo in the readme. Feel free to change it, but it would be great to leave one.